python/version_details/version_details.py

```diff
@@ -520,6 +520,10 @@ def load_data(self, entity):
             self.clear()
             return
 
+        # Ensure project schema will be loaded. The project schema is needed
+        # to reference Version fields in various UI and other elements.
+        shotgun_globals.run_on_schema_loaded(lambda : None, entity.get("project"))
```

Comment:
Thanks for explaining @eliveikis makes sense to me now and seems correct.

@thebeeland Let us know if you are good with @eliveikis explanation and all your concerns are addressed and give a final approval for merging. Thanks